cmd/main.go

```diff
@@ -77,8 +75,15 @@ func main() {
 		}
 	}
 
+	var branchCacheVersionBucket string
+	if *svobsMode {
+		branchCacheVersionBucket = "datcom-control"
+	} else {
+		branchCacheVersionBucket = "prophet_cache"
```

Comment:
This should be "automation_control", no?

Reply:
That bucket only has the base cache version files: https://pantheon.corp.google.com/storage/browser/automation_control;tab=objects?forceOnBucketsSortingFiltering=false&project=google.com:datcom-store-dev&prefix=&forceOnObjectsSortingFiltering=false

Reply:
Ah, I see!